app/policies/application_policy.rb

```diff
@@ -76,16 +76,16 @@ def same_org?
 
   def is_admin_same_org?
     # eventually everything should use this
-    user.casa_admin? && same_org?
+    user&.casa_admin? && same_org?
   end
 
   def is_supervisor?
-    user.supervisor?
+    user&.supervisor?
   end
 
   def is_supervisor_same_org?
     # eventually everything should use this
-    user.supervisor? && same_org?
+    is_supervisor? && same_org?
```

Comment:
Could move these changes out to CaseGroupPolicy if don't want to tolerate nil user in AppPolicy (currently give NoMethodErrors)